cas_server/views.py

```diff
@@ -626,6 +626,23 @@ def process_get(self):
         """
         # generate a new LT
         self.gen_lt()
+        if settings.CAS_DJANGO_AUTOLOGIN and self.request.user.is_authenticated:
+            self.request.session.set_expiry(0)
+            self.request.session["username"] = self.request.user.username
+            self.request.session["warn"] = False
+            self.request.session["authenticated"] = True
+            self.renewed = True
```

Comment:
I think this is not OK as you do not know if the user renew its authentication or not. In fact, I'm not sure how we can make the "authentication renewal" mechanism work by using directly the Django auth...